src/components/teams/form/Toolbar.js

```diff
@@ -79,6 +83,20 @@ function EditTeamToolbar(props) {
                         {t("leave")}
                     </Button>
                 )}
+                {!isAdmin && !isMember && (
```

Comment:
This boolean can be extracted into a `const` and then reused below. 
`const notInTeam = !isAdmin && !isMember`;